src/org/isf/medicalstockward/service/MedicalStockWardIoOperationRepository.java

```diff
@@ -13,29 +13,26 @@
 @Repository
```

Comment:
it's not needed on jpa repository interfaces

Reply:
Hi @tomek, we had a specific tasks for that, are you sure that Spring will not complain in model mapping?
https://openhospital.atlassian.net/browse/OP-102

Reply:
So I am not sure if I should remove those annotations or not. But I think if it is possible to remove them, then they should be removed from ALL repositories and not just from the Repositories used in classes of this specific ticket. So I tend to leave it as is in the course of this PR. @mwithi maybe this could be reviewed in the course of the mentioned ticket OP-102?

Reply:
Yes I agree @marcoebbinghaus, I think we can leave it as is at the moment, and discuss the @tomek suggestion on the OP-102 page and, in case, start a new ticket for the overall code base change.